source/evaporation/evaporation_mass_flux_operator_interface_value.cpp

```diff
@@ -112,32 +124,35 @@ namespace MeltPoolDG::Evaporation
     /*
      * get temperature values at evaluation points
      */
-    Utilities::MPI::RemotePointEvaluation<dim, dim> cache;
+    Utilities::MPI::RemotePointEvaluation<dim, dim> remote_point_evaluation(
+      1e-6 /*tolerance*/, true /*unique mapping*/);
+
+    remote_point_evaluation.reinit(evaluation_points,
+                                   scratch_data.get_triangulation(),
+                                   scratch_data.get_mapping());
 
     temperature.update_ghost_values();
+
     const auto temperature_evaluation_values =
-      dealii::VectorTools::point_values<1>(scratch_data.get_mapping(),
+      dealii::VectorTools::point_values<1>(remote_point_evaluation,
                                            scratch_data.get_dof_handler(temp_dof_idx),
-                                           temperature,
-                                           evaluation_points,
-                                           cache,
-                                           dealii::VectorTools::EvaluationFlags::max);
+                                           temperature);
     temperature.zero_out_ghost_values();
 
     Assert(temperature_evaluation_values.size() == evaluation_points.size(),
            ExcMessage("The size of vectors must match."));
 
     /*
-     * compute evaporative mass flux
+     * compute evaporative mass flux from the temperature value at the interface
      */
     evaporative_mass_flux.zero_out_ghost_values();
     evaporative_mass_flux = 0.0;
 
-    for (unsigned int i = 0; i < evaluation_points.size(); i++)
+    for (unsigned int i = 0; i < evaluation_points.size(); ++i)
       {
-        evaporative_mass_flux[temp_local_dof_indices[i]] =
+        evaporative_mass_flux[dof_indices[i]] =
           evaporation_model.local_compute_evaporative_mass_flux(
-            temperature[temp_local_dof_indices[i]]);
+            temperature_evaluation_values[dof_indices[i]]);
```

Comment:
```suggestion
            temperature_evaluation_values[i]);
```
Shouldn't this be like this? `temperature_evaluation_values` is returned by PRE.

Reply:
Eagle eyes, true! I already integrated into the current status, where I included the number of iterations. Commit follows asap.